libtiny_client/src/state.rs

```diff
@@ -88,7 +88,7 @@ struct StateInner {
     ///
     /// TODO: I'm not sure if this is necessary. Why not just create channel tabs in the specified
     /// order, in TUI?
-    chans: Vec<(String, HashSet<String>)>,
+    chans: Vec<Chan>,
```

Comment:
I thought this would be a nice cleanup and allow me to hold the number of retries available per channel.